frontend/src/metabase/visualizations/visualizations/SmartScalar.jsx

```diff
@@ -0,0 +1,196 @@
+import React from "react";
+import { Box, Flex } from "grid-styled";
+import { t, jt } from "c-3po";
+import _ from "underscore";
+
+import { formatNumber, formatValue } from "metabase/lib/formatting";
+import colors from "metabase/lib/colors";
+
+import Icon from "metabase/components/Icon";
+
+import * as Query from "metabase/lib/query/query";
+import * as Card from "metabase/meta/Card";
+import { parseFieldBucketing, formatBucketing } from "metabase/lib/query_time";
+
+import { columnSettings } from "metabase/visualizations/lib/settings/column";
+import { NoBreakoutError } from "metabase/visualizations/lib/errors";
+
+import ScalarValue, {
+  ScalarWrapper,
+  ScalarTitle,
+} from "metabase/visualizations/components/ScalarValue";
+
+export default class Smart extends React.Component {
+  static uiName = "Smart Scalar";
```

Comment:
Do we want to call it a "Smart Number" to match the "Number" viz name? "Scalar" is a more technical term than we usually use.